packages/core/src/process-request.ts

```diff
@@ -269,24 +271,28 @@ export const processRequest = async <TContext = {}, TRootValue = {}>(
               headers: [],
               payload: formatPayload({
                 payload: result,
-                context,
+                contextValue,
                 rootValue,
                 document,
                 operation,
               }),
             };
           }
         }
-      } catch (executionError) {
-        throw new HttpError(
-          500,
-          "Unexpected error encountered while executing GraphQL request.",
-          {
-            graphqlErrors: [new GraphQLError(executionError.message)],
-          }
-        );
+      } catch (executionError: any) {
```

Comment:
```suggestion
      } catch (executionError: unknown) {
```

Might be the better default